# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/performance/PerformanceWorker.cpp
# Commit: 5598943a94fd
# Full Hash: 5598943a94fd9e6be45bbc7125ef7bac49a4bf05
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2021-12-15 21:51:13
# Regressor Bug: 1744025
# File Overlap Count: 3
# Description:
#   Bug 1744025: Use CheckedUnsafePtr<WorkerPrivate> in WorkerGlobalScopeBase. r=dom-worker-reviewers,asuth
#   
#   Depends on D132708
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132709
# ==============================================================================

diff -r 43186fbbf8b4 -r 5598943a94fd dom/performance/PerformanceWorker.cpp
--- a/dom/performance/PerformanceWorker.cpp	Wed Dec 15 13:48:17 2021 +0000
+++ b/dom/performance/PerformanceWorker.cpp	Wed Dec 15 13:48:17 2021 +0000
@@ -5,7 +5,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "PerformanceWorker.h"
-#include "mozilla/dom/WorkerPrivate.h"
+#include "mozilla/dom/WorkerScope.h"
 #include "mozilla/StaticPrefs_dom.h"
 
 namespace mozilla::dom {